tests/components/application-connector/docs/compass-runtime-agent-tests.md

```diff
@@ -0,0 +1,135 @@
+
+![Compass Runtime Agent tests architecture](assets/compass-runtime-agent-tests-architecture.svg)
+
+The interactions between components are the following:
+
+1. Compass Runtime Agent periodically fetches certificates from Compass Connector.
+2. Compass Runtime Agent periodically fetches applications from Compass Director.
+3. Compass Runtime Agent Test sends GraphQL mutations to Compass Director to create, modify, or delete Applications.
+4. Compass Runtime Agent Test verifies whether corresponding Application CRs were created, modified, or deleted.
+5. Compass Runtime Agent Test verifies whether secret with certificates used for communication with Director was created.
+6. Compass Runtime Agent Test verifies whether secret with CA root certificate used by Istio Gateway was created.
+7. Compass Runtime Agent Test verifies CompassConnection CR contents.
+
+## Building
+
+Pipelines build the Compass Runtime Agent test using the **release** target from the `Makefile`.
+
+To build **and push** the Docker images of the tests, run:
+
+``` sh
+./scripts/local-build.sh {DOCKER_TAG} {DOCKER_PUSH_REPOSITORY}
+```
+
+This will build the following images:
+- `{DOCKER_PUSH_REPOSITORY}/compass-runtime-agent-test:{DOCKER_TAG}`
+
+## Running
+
+Tests can be run on any Kyma cluster with Compass Runtime Agent.
+
+Pipelines run the tests using the **test-compass-runtime-agent** target from the `Makefile`.
+
+### Deploy a Kyma cluster locally
+
+1. Provision a local Kubernetes cluster with k3d:
+   ```sh
+   kyma provision k3d
+   ```
+
+2. Install the minimal set of components required to run Compass Runtime Agent **for Kyma SKR (Compass mode)**:
+
+    ```bash
+    kyma deploy --components-file ./resources/installation-config/mini-kyma-skr.yaml --value global.disableLegacyConnectivity=true
+    ```
+
+   >**TIP:** Read more about Kyma installation in the [official Kyma documentation](https://kyma-project.io/docs/kyma/latest/02-get-started/01-quick-install/#install-kyma).
+
+### Test setup - Compass Runtime Agent configuration
+
+[The values.yaml](../resources/charts/compass-runtime-agent-test/values.yaml) file contains environment variables that are used in Compass tests. These values can be modified as needed.
+
+- `APP_DIRECTOR_URL` - Compass Director URL
+- `APP_TESTING_TENANT` - Tenant used in GraphQL calls
+- `APP_SKIP_DIRECTOR_CERT_VERIFICATION` - Skip certificate verification on the Director side
+- `APP_OAUTH_CREDENTIALS_SECRET_NAME` - Secret name for Compass OAuth credentials
+- `APP_OAUTH_CREDENTIALS_NAMESPACE` - Namespace for Compass OAuth credentials
+
+### Run the tests
+
+1. Before running the test export the following environment variables
+- `COMPASS_HOST` - host running Compass 
+- `COMPASS_CLIENT_ID` - client id used for fetching authorisation tokens
+- `COMPASS_CLIENT_SECRET` - client secret used for fetching authorisation tokens 
```

Comment:
```suggestion
- `COMPASS_CLIENT_SECRET` - client Secret used for fetching authorization tokens 
```

Reply:
I edited the suggestion to bold it instead of formatting as code, as this is a parameter. See the comment above.
Also, it falls under point 1, so it should be indented.